README.md

```diff
@@ -519,6 +519,9 @@ lora_modules_to_save:
 #  - lm_head
 lora_out_dir:
 lora_fan_in_fan_out: false
+ia3_target_modules:  # target modules for IA3, for llama, k, v, and down projections
+ia3_feedforward_modules:  # ffn modules for IA3, for llama down projection
+ia3_fan_in_fan_out:
```

Comment:
The `target modules` and `fan in fan out` feels a bit redundant since we have two similar names..